### ISSUE TITLE
What is the best practice to save/load model with inputs (as named features) and labeled outputs?

### ISSUE THREAD

**Issue by acriptis** [fchollet/keras#4228]
I need to save a neural network model with names of input features and names of outputs. So I need to save a config, weights and two lists: 
- input_features_labels (list of str, ex.: ["length_cm", "diameter_cm", "weight_g_normalized"])
- output_labels (list of str, ex.: ["age_prediction", "gender_prediction"])

I found very useful method `keras.models.save_model(model, filepath)` but it seems I can not put there my own arguments (two lists with labels) and save all information into one file.

Maybe is it possible to specify input and output labels within `Sequential` Model, so I could restore model from file with also restoring exact list of names of input features and outputs? I found that sometimes keras uses concept `weight_names` but I haven't figured out how to set it for input and output layers so it would be restorable. 

Any recommendations for solving this problem?
